Palace/AppInfrastructure/TPPAppTheme.swift

```diff
@@ -25,7 +25,7 @@
       return colorFromHex(hex(theme))
     } else {
       Log.error(#file, "Given theme color is not supported: \(name)")
-      return .black
+      return UIApplication.darkModeEnabled ? .white : .black
```

Comment:
(added description for iconColor below) when iOS changes dark mode value, text remains the same color it was set when the app started.

![Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2021-07-13 at 00 54 30](https://user-images.githubusercontent.com/941531/125347871-eacdcf00-e374-11eb-856e-199f53454f2e.png)